recorder/src/main/cpp/config.cc

```diff
@@ -101,7 +101,10 @@ void ConfigurationOptions::load(const char* options) {
                     noctx_cov_pct = 100;
                 }
             } else if (strstr(key, "allow_sigprof") == key) {
-                allow_sigprof = (std::string(value) == "y") || (std::string(value) == "Y");
+                auto ch = value[0];
```

Comment:
this will result in any string starting with "y"/"Y" result in Yes. Do we want that?